contracts/src/warpDiscordBot/messagesContent/write/removeReaction.ts

```diff
@@ -0,0 +1,29 @@
+import { validateInputArgumentPresence, validateString } from '../../../utils';
+import { ContractAction, ContractState, ContractResult } from '../../types/types';
+import { countBoostsPoints } from './addMessage';
+import { subtractTokensBalance } from './removeMessage';
+
+declare const ContractError;
+declare const SmartWeave;
+
+export const removeReaction = async (
+  state: ContractState,
+  { input: { id, roles } }: ContractAction
+): Promise<ContractResult> => {
+  validateInputArgumentPresence(id, 'id');
+  validateString(id, 'id');
+
+  const counter = state.counter[id];
+  let boostsPoints = state.reactionsTokenWeight;
+  boostsPoints *= countBoostsPoints(state, counter.boosts, roles);
```

Comment:
hmm, should the 'remove' function user the same exact boost that was used when the points were added?
since the boosts can change over time, I believe that the 'remove' function can be applied with different boosts than add?